.github/workflows/release.yml

```diff
@@ -51,8 +51,10 @@ jobs:
 
       - name: Build 'kolena' documentation and push to GitHub Pages
         run: |
+          git config user.name 'kolenabot'
```

Comment:
Is this the same entity where we used to configure ssh key in steps above?

Reply:
side question: config here would make tracking better. However, technically we can protect this branch without these username/email config extras, right?

Reply:
> technically we can protect this branch without these username/email config extras

What protections do you have in mind? I've set up protections on `release/docs` to only allow commits from kolenabot.

Reply:
Same as you have setup. I'm just curious since ssh-key is used to push to the branch, should be able to verify identity with `git config user.name` etc.

Pure discussion, the PR is good in correctness.

Reply:
Ah, good question (related to behavior mentioned here: https://github.com/kolenaIO/kolena/pull/80#issuecomment-1593407357).

I haven't been able to find relevant documentation about what protections GitHub actually applies when you restrict branches to certain users. I have to believe that they use the underlying creds/SSH key, but also wouldn't be entirely surprised if it somehow relied on the author information. After this PR lands, both will be set up for kolenabot.